docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,724 @@
+}
+
+module.exports.createWorkloadModule = createWorkloadModule;
+```
+
+### initializeWorkloadModule
+
+This method is used to prepare any items required by the `submitTransaction` method as the benchmark completes. 
+
+The number of assets to be created will be given as `roundArguments.assets`. We create assets using the smart contract by populating an arguments object, which defines the transaction body, and using the Caliper API `sendRequests`. Note the `readOnly` value passed within the arguments.
+
+The method should look like this:
+```bash
+    async initializeWorkloadModule(workerIndex, totalWorkers, roundIndex, roundArguments, sutAdapter, sutContext) {
+        await super.initializeWorkloadModule(workerIndex, totalWorkers, roundIndex, roundArguments, sutAdapter, sutContext);
+
+        const requests = [];
+        for (let i=0; i<this.roundArguments.assets; i++) {
+            const assetID = `${this.workerIndex}_${i}`;
+            console.log(`Client ${this.workerIndex}: Creating asset ${assetID}`);
+            const args = {
+                contractID: this.roundArguments.contractID,
+                contractFunction: 'createCar',
+                invokerIdentity: 'Admin@org1.example.com',
+                contractArguments: [assetID,'blue','ford','focus','jim'],
+                readOnly: false
+            };
+
+            requests.push(args);
+        }
+
+        try {
+            await this.sutAdapter.sendRequests(requests);
+        } catch (error) {
+            console.log(`Client ${this.workerIndex}: Smart Contract threw with error: ${error}` );
+        }
+    }
+```
+
+In the above example, different assets will be created that have the same parameters (blue, ford, focus, Jim). Comparing the above to the smart contract method itself, it should be evident that there is a 1:1 mapping of contract arguments to the method parameters.
+
+### submitTransaction
+
+This function runs repeatedly in the benchmark test phase. We will be evaluating the `queryCar` smart contract function by querying the assets we created in the `initializeWorkloadModule` method. 
+
+First, create a string identity for the asset to query, formed by the concatenation of the worker index and a random integer between 0 and the number of created assets. 
+
+Then return the call on `querySmartContract`, passing context, contract name, contract version, and an object that contains: `chaincodeFunction` set as `queryCar`; `invokerIdentity` set as `admin@org1.example.com`; and `chaincodeArguments` set as an array that contains the asset to query in this run. 
+
+The function should look like this:
+```bash
+    async submitTransaction() {
+        const randomId = Math.floor(Math.random()*this.roundArguments.assets);
+        const myArgs = {
+            contractID: this.roundArguments.contractID,
+            contractFunction: 'queryCar',
+            invokerIdentity: 'Admin@org1.example.com',
+            contractArguments: [`${clientIdx}_${randomId}`],
```

Comment:
clientIdx isn't defined, maybe it should be this.workerIndex ?

Reply:
clientIdx still not defined :-)

Reply:
I have a brain, I promise 🧠